<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Resimler</string>
    </property>
    <widget class="QPushButton" name="btnEiffel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Eiffel Tower</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnArc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Arc de Triomph</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPisa">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Pisa Tower</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>231</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Bilgiler</string>
    </property>
    <widget class="QPushButton" name="btnEiffelBilgi">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Eiffel Tower</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPisaBilgi">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Pisa Tower</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnArcBilgi">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Arc de Triomph</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>515</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
